ops/jujulog.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
Can we please name this file as "log.py" instead?  We should be able to put other logging things here as well if necessary, instead of creating another one.

Reply:
OK, I'll `git mv` that and `test_jujulog.py`.

Reply:
https://github.com/canonical/operator/pull/133/commits/2c778fc3f128b088790229bc3cc55bf7f7daedd6